.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
 
-.env
-.env
+.DS_Store
```

Comment:
this is OS-specific and should just be in an individual's local `~/.gitignore_global`. I'm removing all these DS_Stores in #18